src/Components/Comments.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import axios from 'axios';
+import UnfoldMoreDoubleIcon from '@mui/icons-material/UnfoldMoreDouble';
+
+
+export default class Comments extends React.Component {
+    constructor(props) {
+    super(props);
+  
+    // Initializing the state 
+    this.state = {
+        renderType:'comments',
+        items: [],
+        revData : false
+      };
+  }
+
+  
+  render() {
+    const Comments = this.props.items;
```

Comment:
no need to assign props.items into variable and then use , use directly with combination of && operator